app/views/admin/roles/_form.html.erb

```diff
@@ -5,48 +5,116 @@
 
 <%= form_for role, as: :role, url: role_url_for(role) do |form| %>
   <%= form.errors %>
+    <%= render "govuk_publishing_components/components/input", {
+        label: {
+          text: "Role title"
+        },
+        value: role.name,
+        name: "role[name]",
+        id: "role_name",
+        type: "text"}
+    %>
 
-  <fieldset>
-    <%= form.text_field :name, label_text: 'Role title' %>
+    <%= render "govuk_publishing_components/components/select", {
+      id: "role_type_id",
+      label: "Role type",
+      name: "role[role_type]",
+      options:
+        [{value: nil, text: nil}] + role_type_options.map do |group, name_vs_type|
+          name_vs_type.map do |name, type|
+            {text: name, value: type, selected: role.role_type == type }
+          end
+        end.flatten
+    } %>
 
-    <div class="form-group">
-      <%= form.label :role_type %>
-      <%= form.select :role_type, grouped_options_for_select(role_type_options, role.role_type), { include_blank: true }, {class: 'form-control input-md-3'} %>
-    </div>
-    <div class="form-group">
-      <%= form.label :organisation_ids, 'Organisations' %>
-      <%= form.select :organisation_ids, options_from_collection_for_select(Organisation.with_translations(:en), 'id', 'select_name', role.organisation_ids), {}, multiple: true, class: 'chzn-select form-control', data: { placeholder: "Choose organisations…" } %>
-    </div>
-    <div class="form-group">
-      <%= form.label :whip_organisation_id, 'Whip Organisation' %>
-      <%= form.select :whip_organisation_id, options_from_collection_for_select(Whitehall::WhipOrganisation.all, 'id', 'name', role.whip_organisation_id),  { include_blank: true }, {class: 'form-control input-md-3'} %>
-    </div>
-    <div class="form-group">
-      <%= form.label :role_payment_type_id, 'Payment options' %>
-      <%= form.select :role_payment_type_id, options_from_collection_for_select(RolePaymentType.all, :id, :name, form.object.role_payment_type_id), { include_blank: true }, {class: 'form-control input-md-3'}%>
-    </div>
-    <div class="form-group">
-      <%= form.label :attends_cabinet_type_id, 'Attends cabinet options' %>
-      <%= form.select :attends_cabinet_type_id, options_from_collection_for_select(RoleAttendsCabinetType.all, :id, :name, form.object.attends_cabinet_type_id), { include_blank: true }, {class: 'form-control input-md-3'} %>
-    </div>
-    <div class="form-group">
-      <%= form.label :worldwide_organisation_ids, 'Worldwide organisations' %>
-      <%= form.select :worldwide_organisation_ids, options_from_collection_for_select(WorldwideOrganisation.all, 'id', 'name', role.worldwide_organisation_ids), {}, multiple: true, class: 'chzn-select form-control', data: { placeholder: "Choose worldwide organisations…" } %>
-    </div>
-  </fieldset>
+    <%= render "components/autocomplete", {
+      id: "role_organisation_ids",
+      label: {text: "Organisations"},
```

Comment:
This one is super nitty sorry! But we have a pattern of moving hashes within components over multiple lines

```
label: { 
  text: "Organisations"
},
```
```